src/translations/ca.translation

```diff
@@ -40,6 +40,10 @@
         <label>Compte</label>
         <name>DefaultAccountName</name>
     </customLabels>
+    <customLabels>
+        <label><!-- We can’t find the Administrative Account record type in your org. Following installation, use Object Manager to create a new record type named ‘Administrative’ for the Account object. --></label>
```

Comment:
@MatthewBlanski did you mean to comment this out for the label?

Reply:
@jofsky This is actually how the metadata API pulls it down. I tend to not touch it.

Reply:
Oh interesting!